### DISCUSSION TITLE
Missing contribution instructions?

### DISCUSSION THREAD

**Discussion by blaemmle** [DependencyTrack/dependency-track#1147] [Q&A]
Hi, 
I searched for an instruction about how to contribute, but i cant find anything in the [documentation](https://docs.dependencytrack.org/) or as an CONTRIBUTING.md file in the Repository. 
Same goes for the frontend Repository. 

I only found this: 
https://groups.io/g/dependency-track/topic/dco_now_enforced/81583987?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,81583987

Did i missed it or is their nothing available?
Thanks in advance!

**Comment by stevespringett** [comment]
Hi @blaemmle. Currently the project does not have a CONTRIBUTING.md file in place but likely should. I'll likely start with https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md as a foundation and modify it to be specific to the Dependency-Track project.

**Reply by yangsec888** [reply]
I have a PR in progress and looking for the same thing.
